tests/spread/general/classic-patchelf-override-multi-parts/snap/snapcraft.yaml

```diff
@@ -0,0 +1,36 @@
+name: simple-snap
+version: "1.0"
+summary: Build a simple classic snap with override-prime
+description: |
+  Build a simple classic snap with override-prime that tests patchelf
+
+base: core22
+
+confinement: classic
+
+apps:
+  hello:
+    command: bin/hello
+
+parts:
+  hello:
+    source: ../hello
+    plugin: make
+    build-packages:
+      - gcc
+      - libc-dev
+      - libcurl4-openssl-dev
+    stage-packages: [libcurl4]
+    override-prime: |
+      craftctl default
+
+  hello2:
+    source: ../hello2
+    plugin: make
+    build-attributes:
+      - enable-patchelf
+    build-packages:
+      - gcc
+      - libc-dev
+      - libcurl4-openssl-dev
+    stage-packages: [libcurl4]
```

Comment:
Based on this success & failure, what happens with multiple parts that all override prime?